docs/app/Examples/collections/Form/Validation/FormValidatingOnBlurAndOtherEventsExample.js

```diff
@@ -1,73 +1,40 @@
-import React, { Component } from 'react'
-import { Button, Checkbox, Field, Fields, Form, Input, Message } from 'stardust'
+import React from 'react'
+import { Button, Checkbox, Field, Fields, Form, Input } from 'stardust'
 
-export default class FormValidatingOnBlurAndOtherEventsExample extends Component {
-  formSettings = {
-    inline: true,
-    on: 'blur',
-    fields: {
-      'first-name': {
-        identifier: 'first-name',
-        rules: [
-          { type: 'empty' },
-        ],
-      },
-      'last-name': {
-        identifier: 'last-name',
-        rules: [
-          { type: 'empty' },
-        ],
-      },
-      'username': {
-        identifier: 'username',
-        rules: [
-          { type: 'empty' },
-        ],
-      },
-      'password': {
-        identifier: 'password',
-        rules: [
-          { type: 'empty' },
-          { type: 'minLength[6]' },
-        ],
-      },
-      'terms': {
-        identifier: 'terms',
-        rules: [
-          { type: 'checked' },
-        ],
-      },
-    },
-  };
-
-  render() {
-    return (
-      <Form settings={this.formSettings} className='segment'>
-        <p>Let's go ahead and get you signed up.</p>
-        <Fields evenlyDivided>
-          <Field>
-            <label>First Name</label>
-            <Input placeholder='First Name' name='first-name' type='text' />
-          </Field>
-          <Field>
-            <label>Last Name</label>
-            <Input placeholder='Last Name' name='last-name' type='text' />
-          </Field>
-        </Fields>
-        <Fields evenlyDivided>
-          <Field label='Username'>
-            <Input placeholder='Username' name='username' type='text' />
-          </Field>
-          <Field label='Password'>
-            <Input type='password' name='password' />
-          </Field>
-        </Fields>
-        <Field className='inline'>
-          <Checkbox name='terms' className='hidden' label='I agree to the terms and conditions' />
-        </Field>
-        <Button className='blue submit'>Submit</Button>
-        <Message className='error' />
-      </Form>
-    )
-  }
+const fields = {
+  firstName: 'empty',
+  lastName: 'empty',
+  username: 'empty',
+  password: 'minLength[6]',
+  terms: 'checked',
 }
+
+const FormValidatingOnBlurAndOtherEventsExample = (props) => (
+  <Form className='segment' fields={fields} on='blur' inline onSuccess={() => false}>
```

Comment:
This is a great question.  I went back and forth with this actually.  I actually committed a `defaultProps` function that returned false and then reverted it.  I considered our current approach:

Stardust...

1. is a React port of Semantic UI
1. includes some addons and extra features made possible by JS components
1. is as close to vanilla React and Semantic UI as possible
  1. never sets default props
  1. never overrides styles
  1. spreads your props on the element of concern (`<input />` for `<Input />`)

I opted for consistency here in not defaulting.  I figured this is the "Semantic UI way" of doing things.  Also, I'm not sure that everyone's use case is to prevent submit.  For instance, Semantic UI actually disables submit automatically for you if you use their API setting.  I'm still on the fence with our default though.

What are your thoughts on keeping with some of this ^ vs defaulting in this one instance for our use cases?